### PR TITLE
My portofolio: works section

### PR DESCRIPTION
**Pull request by sarazaToussaint**
In this project we:

- Added Makeup for the works section
-  Added the layout of the works section
- Added icons in the project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -43,6 +40,127 @@ <h1>I'm Saraza<br> Glad to see<br> you!</h1>
...
+         </ul>
+         <input type="button" value="Get my resume">
+       </div>
+       <div class="works">
+           <h4>Languages</h4>
+           <ul>
+               <li class="lg"><img src="./images/html.svg" alt="html"><p>HTML</p></li>
+               <li class="lg"><img src="./images/css.svg" alt="css"><p>CSS</p></li>
+               <li class="lg"><img src="./images/js.png" alt="javascript"><p>Javascript</p></li>
+           </ul>
```
Comment (vic778) [inline index.html:157]: kindly use the vector icon for this section. when the user clicks on the title the vector down must appear and the unopened title must contain the vector right as it is well demonstrated in the design of the [Figma](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/images/m1_setup/about-myself-mobile.png) template.

File: index.html [hunk 2]
```diff
@@ -43,6 +40,127 @@ <h1>I'm Saraza<br> Glad to see<br> you!</h1>
...
+           <ul class="fram">
+            <li>Frameworks</li>
+            <li>Skills</li>
+           </ul>
```
Comment (vic778) [inline index.html:161]: kindly this section should show the right vector because the user is not clicking on the title to see the skills

**Review by vic778 — CHANGES_REQUESTED** [review]
### STATUS: CHANGES REQUIRED :yellow_circle: 
Hi @sarazaToussaint , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### Project highlights:
- [x] All linter checks are passing :heavy_check_mark: 
- [x] Descriptive commits and correct use of the Git Flow :fire:
- [x] :white_check_mark: page titles :white_check_mark: text headings :white_check_mark: basic structure of the page 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


so close

Cheers and Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Hamzaoutdoors — APPROVED** [review]
Hi @sarazaToussaint, 

## Congratulations! 🎉 Project Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚀

To Highlight:

- [x] The Pull Request has a proper title and description. ✔️
- [x] You have added a descriptive Readme file. Good Job. ✔️
- [x] There is no linter error is present. ✔️
- [x] Project has met all the technical aspects you needed for this project! ✔️ 

_Cheers and Happy coding and Keep rocking!_ 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**